modules/zip/src/parse-zip/zip-compozition.ts

```diff
@@ -1,8 +1,13 @@
 import {FileHandleFile, concatenateArrayBuffers} from '@loaders.gl/loader-utils';
-import {ZipEoCDRecord, parseEoCDRecord, updateEoCD} from './end-of-central-directory';
+import {ZipEoCDRecord, generateEoCD, parseEoCDRecord, updateEoCD} from './end-of-central-directory';
 import {CRC32Hash} from '@loaders.gl/crypto';
 import {generateLocalHeader} from './local-file-header';
 import {generateCDHeader} from './cd-file-header';
+import path from 'path';
```

Comment:
REQUIRED CHANGE We should not include Node.js imports. While path can be imported in browsers it pulls in a big polyfill. See if you can `import {path} from '@loaders.gl/loader-utils` instead.

Reply:
Corrected